src/command_modules/azure-cli-interactive/azure/cli/command_modules/interactive/tests/test_shell_progress.py

```diff
@@ -24,6 +25,7 @@ def test_shell_progress_write(self):
         self.assertEqual(prog.get_progress_message(), 'Test1')
         self.assertEqual(prog.PROGRESS_BAR, "")
 
+    @unittest.skipIf(os.getenv('TERM') is None, 'Skip when $TERM is missing')
```

Comment:
@oakeyc @derekbekoe when this test is run in the container, the `tput` command will fail. You have any workaround?